$NetBSD: patch-aa,v 1.1.1.1 2008/10/25 00:32:23 bjs Exp $

We install this file in share/doc/libatomic_ops/README_headers.

--- src/atomic_ops/sysdeps/README.orig	2006-07-11 19:29:30.000000000 -0400
+++ src/atomic_ops/sysdeps/README
@@ -1,4 +1,4 @@
-There are two kinds of entities in this directory:
+There are two kinds of entities in <libatomic_ops/sysdeps>:
 
 - Subdirectories corresponding to specific compilers (or compiler/OS combinations).
   Each of these includes one or more architecture-specific headers.
